config/locales/en.yml

```diff
@@ -57,9 +58,12 @@ en:
     read_next: "Read next"
     reply: "Reply"
     report_abuse: "Report abuse"
+    save: Save
```

Comment:
While it's strictly speaking not necessary, we opted for quoting all strings in our translation files, please keep it this way. This also avoids some issues with YAML's implicit typing, e.g. 

```ruby
YAML.load("- yes\n- maybe\n- no")
#=> [true, "maybe", false]
```